group_vars/tcbsd_plcs/vars.yml

```diff
@@ -0,0 +1,36 @@
+---
+ansible_user: Administrator
+ansible_ssh_private_key_file: /Users/klauer/Repos/twincat-bsd-ansible-testing/tcbsd_key_rsa
```

Comment:
Does this (and the rest of the file, for that matter) get templated / filled in some way? Or is this file just meant to be an example?

Reply:
A good catch - just an example (still probably something I should have changed before asking for a review to be honest...)